sphinx/environment/__init__.py

```diff
@@ -61,7 +61,7 @@
 
 # This is increased every time an environment attribute is added
 # or changed to properly invalidate pickle files.
-ENV_VERSION = 56
+ENV_VERSION = 57
```

Comment:
Instead of this, please update `env_version` field which `sphinx.domains.cpp:setup()` returns. It describes the version of domain data structure now.

Reply:
Thanks!